<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>webservicesDialog</class>
 <widget class="QDialog" name="webservicesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWebView" name="Webview">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>600</width>
     <height>360</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QPushButton" name="openInBrowser">
   <property name="geometry">
    <rect>
     <x>489</x>
     <y>2</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>open in browser</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zoomPlus">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>360</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Zoom +</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="urlLine">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>2</y>
     <width>249</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="zoomMinus">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>360</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Zoom -</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openGeoJSON">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>2</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Open GeoJSON</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
